src/slogger/slogger.cc

```diff
@@ -20,14 +20,18 @@
 
 #include "common/platform.h"
 #include "slogger/slogger.h"
+#include "config/cfg.h"
 
 #include <errno.h>
 #include <stdarg.h>
 #include <stdio.h>
 #include <unistd.h>
+#include <boost/algorithm/string/case_conv.hpp>
```

Comment:
The std already contains algorithms for case conversions. Please consider them to avoid extra dependencies.